Sources/Quick/Example.swift

```diff
@@ -71,7 +71,11 @@ final public class Example: NSObject {
         }
         group!.phase = .beforesFinished
 
-        closure()
+        do {
+            try closure()
+        } catch {
+            print("Test \(name) threw unexpected error: \(error.localizedDescription)")
```

Comment:
Hey @marcelofabri, I think the `Callsite` class will probably need to be refactored to use `StaticString` to use `XCTFail` as probably the best way to go